src/ast_traverse.mli

```diff
@@ -54,7 +54,7 @@ end
 
 class map_with_path : [string] map_with_context
 
-class map_with_code_path : [Code_path.t] map_with_context
+class map_with_expansion_ctxt_base : [Expansion_context.Base.t] map_with_context
```

Comment:
I suggest to call this `map_with_expansion_context`. I'm not sure the `_base` suffix is adding much and I find it slightly odd to mix full words and abbreviations.

Reply:
I agree, I'll fix that!